package.json

```diff
@@ -50,7 +50,7 @@
     "@types/w3c-image-capture": "^1.0.10",
     "@typescript-eslint/eslint-plugin": "^6.9.1",
     "@typescript-eslint/parser": "^6.9.1",
-    "@webgpu/types": "^0.1.46",
```

Comment:
To fix the build, run `npm install` locally to update the `package-lock.json`, and commit the result.

Reply:
Done